todolist/public/index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
```suggestion
<html lang="ko">
```
사소하지만 `ko`로 변경하는게 좋을 것 같습니다